apps/interactor/interactor/tasks/registered/circadian/__init__.py

```diff
@@ -0,0 +1,60 @@
+from interactor.tasks.registered.circadian.options import Options
+from interactor.tasks.register import registerer
+
+from photons_control.script import FromGenerator
+from photons_messages import LightMessages
+
+from datetime import datetime
+
+
+@registerer
+def register(tasks):
+    tasks.register("circadian", Options, run)
+
+
+async def run(final_future, options, progress):
+    async def action(reference, sender, **kwargs):
+        now = datetime.now()
+        if int(now.strftime("%w")) not in options.days:
+            progress("Skipping day", now=now.strftime("%w"), only_for=options.days)
```

Comment:
this function will add messages to any of the websockets listening in. It's the same syntax as hp.lc